src/Chunk.ts

```diff
@@ -341,6 +342,9 @@ export default class Chunk {
 		if (this.renderedHash) return this.renderedHash;
 		if (!this.renderedSource) return '';
 		const hash = sha256();
+		if (this.augmentedHash) {
+			hash.update(this.augmentedHash);
```

Comment:
As the `renderedHash` is effectively cached and determined at most once, but sometimes not at all, i.e. if there is no `[hash]` in the name pattern, I wondered if it would not be better to generate this value on the fly. The only necessary parameter `graph` is available on each chunk anyway and we would not need the additional property on the `Chunk` class.